src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_helpers.cc

```diff
@@ -30,12 +30,17 @@ const std::vector<std::string> RCHelpers::buttons_radio() {
   data.push_back(enums_value::kVolumeUp);
   data.push_back(enums_value::kVolumeDown);
   data.push_back(enums_value::kEject);
-  data.push_back(enums_value::kSource);
   data.push_back(enums_value::kShuffle);
   data.push_back(enums_value::kRepeat);
   return data;
 }
 
+const std::vector<std::string> RCHelpers::buttons_audio() {
+  std::vector<std::string> data;
+  data.push_back(enums_value::kSource);
+  return data;
+}
+
```

Comment:
```suggestion
  data.push_back(enums_value::kSource);
  data.push_back(enums_value::kShuffle);
  data.push_back(enums_value::kRepeat);
  return data;
}

```
Gotta revert this, the `SOURCE` button is tied to the `RADIO` module, not the `AUDIO` module (for however odd that is design-wise).

Reply:
Ah I see. Reverted the previous commit